src/main/java/com/powsybl/network/hypothesis/StartupType.java

```diff
@@ -0,0 +1,16 @@
+/**
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.network.hypothesis;
+
+/**
+ * @author Chamseddine BENHAMED <chamseddine.benhamed at rte-france.com>
+ */
```

Comment:
Could you document here in the javadoc what `EMPIL_ECO` and `POURCENT_PIMP` refer to?